src/main/java/com/palechip/hudpixelmod/HudPixelConfig.java

```diff
@@ -137,13 +141,14 @@ public Configuration getConfigFile() {
         return config;
     }
 
-    // for the config gui introduced in 2.1.0, most properties need to be renamed
+    // for the config gui introduced in 2.1.0, most properties need to be
+    // renamed
```

Comment:
Did you auto-format this file? It's probably a setting to break comments after 80 chars. However my code style doesn't follow this. For this reason, I'll revert the comments when I merge this.